sound/soc/sof/intel/hda-dsp.c

```diff
@@ -20,6 +20,17 @@
 #include "../ops.h"
 #include "hda.h"
 
+#define SOF_PM_PG_RSVD		BIT(0)
+
+/* Indicates whether streaming is active */
+#define SOF_PM_PG_STREAMING	BIT(1)
+
+/* Prevent power gating (0 - deep power state transitions allowed) */
+#define SOF_PM_PPG		BIT(2)
+
+/* Prevent clock gating (0 - cg allowed, 1 - DSP clock always on) */
+#define SOF_PM_PCG		BIT(3)
+
```

Comment:
@keyonjie why do we need these? the cavs-header file already has definitions for these. Why not use the same?

Reply:
@ranj063 they could be different, though they are same at the moment, they can diverge later. SOF_PM_xx could be used for other non-cAVS PM_GATE IPCs, so here we are doing mapping.

Reply:
@keyonjie I dont understand. These flags have got to be platform specific isnt it? Why will it change later for the same platform?

Reply:
@ranj063 though we are putting these SOF_PM_xxx in hda-dsp, they are used in FW as general as other SOF IPCs, @mmaka1 correct me if I am wrong.

Reply:
It's fine as long as we clarify in the IPC definition that the flags are platform-specific and the IPC will only be called from platform-specific parts, not by the SOF core.